renderdoc/driver/gl/egl_dispatch_table.h

```diff
@@ -68,6 +68,10 @@ typedef PFNEGLPOSTSUBBUFFERNVPROC PFN_eglPostSubBufferNV;
 typedef PFNEGLSWAPBUFFERSWITHDAMAGEEXTPROC PFN_eglSwapBuffersWithDamageEXT;
 typedef PFNEGLSWAPBUFFERSWITHDAMAGEKHRPROC PFN_eglSwapBuffersWithDamageKHR;
 
+typedef PFNEGLCREATEIMAGEKHRPROC PFN_eglCreateImageKHR;
+typedef PFNEGLDESTROYIMAGEKHRPROC PFN_eglDestroyImageKHR;
+typedef PFNEGLGETNATIVECLIENTBUFFERANDROIDPROC PFN_eglGetNativeClientBufferANDROID;
```

Comment:
@lfedorov I am interested to have your change for our local usage for Android device. But this `PFNEGLGETNATIVECLIENTBUFFERANDROIDPROC` isn't defined in driver/gl/official
/eglext.h. Thus it doesn't compile. How can I get it compiled properly? Thanks.